test/schemas/v0-0-1/tests.schema.json

```diff
@@ -0,0 +1,334 @@
+        },
+        "expCleanSrc": {
+          "$ref": "#/$defs/expCleanSrc"
+        },
+        "expParts": {
+          "$ref": "#/$defs/expParts"
+        },
+        "expErrors": {
+          "$ref": "#/$defs/expErrors"
+        },
+        "only": {
+          "type": "boolean",
+          "description": "Normally not set. A flag to use during development to only run one or more specific tests."
+        }
+      }
+    },
+    "testType": {
+      "description": "The category of test.",
+      "enum": [
+        "syntax",
+        "function"
+      ]
+    },
+    "locale": {
+      "description": "The locale to use for formatting.",
+      "type": "string"
+    },
+    "src": {
+      "description": "The MF2 syntax source.",
+      "$comment": "Array values should be concatenated into a single string. They are only included to help readability in the test JSON.",
+      "type": [
+        "string",
+        "array"
+      ],
+      "items": {
+        "type": "string"
+      }
+    },
+    "params": {
+      "description": "Parameters to pass in to the formatter for resolving external variables.",
+      "type": "array",
+      "items": {
+        "$ref": "#/$defs/var"
+      }
+    },
+    "var": {
+      "type": "object",
+      "oneOf": [
+        {
+          "additionalProperties": false,
+          "required": [
+            "value"
+          ],
+          "properties": {
+            "name": {
+              "type": "string"
+            },
+            "value": {}
+          }
+        },
+        {
+          "required": [
+            "type",
+            "value"
+          ],
+          "properties": {
+            "name": {
+              "type": "string"
+            },
+            "type": {
+              "const": "datetime"
+            },
+            "value": {
+              "type": "string"
+            }
+          }
+        }
+      ]
+    },
+    "exp": {
+      "description": "The expected result of formatting the message to a string.",
+      "type": "string"
+    },
+    "expCleanSrc": {
+      "description": "The expected normalized form of `src`, for testing stringifiers.",
+      "$comment": "Array values should be concatenated into a single string. They are only included to help readability in the test JSON.",
+      "type": [
+        "string",
+        "array"
+      ],
+      "items": {
+        "type": "string"
+      }
+    },
+    "expParts": {
+      "$ref": "formatted-parts.schema.json"
+    },
+    "expErrors": {
+      "description": "The runtime errors expected to be emitted when formatting the message. If expErrors is either absent or empty, the message must be formatted without errors.",
+      "type": [
+        "array",
+        "boolean"
+      ],
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "type"
+        ],
+        "properties": {
+          "type": {
+            "enum": [
+              "bad-input",
```

Comment:
My comment is outside the PR scope.

How do these errors correlate with errors described in [MessageFormat 2.0 Errors](https://github.com/unicode-org/message-format-wg/blob/main/spec/errors.md)?

RangeError (range-error) is bound to EcmaScript implementation.